test/e2e-common.sh

```diff
@@ -73,7 +73,7 @@ readonly EVENTING_KAFKA_SECRET_TEMPLATE="300-kafka-secret.yaml"
 readonly EVENTING_KAFKA_CONFIG_TEMPLATE="300-eventing-kafka-configmap.yaml"
 
 # Strimzi installation config template used for starting up Kafka clusters.
-readonly STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.27.0.yaml"
+readonly STRIMZI_INSTALLATION_CONFIG_TEMPLATE="test/config/100-strimzi-cluster-operator-0.28.0.yaml"
```

Comment:
the PR says 3.1

this just bumps the operator itself

Reply:
Oh, forgot updating it. Thanks for catching it.